app/src/main/res/layout/app_version_fragment.xml

```diff
@@ -24,9 +24,9 @@
         android:layout_height="wrap_content"
         android:background="@drawable/selector_app_version"
         android:fontFamily="sans-serif"
-        android:paddingStart="16dp"
+        android:paddingStart="@dimen/app_version_padding_start"
         android:paddingTop="20dp"
-        android:paddingEnd="16dp"
+        android:paddingEnd="@dimen/app_version_textview_margin_end"
```

Comment:
Rename this to `app_version_text_view_padding_end`

If you are setting padding attribute then the variable name should contain padding in it instead of margin.

Reply:
Also this has been used incorrectly as the value of `app_version_textview_margin_end ` is `28dp` not `16dp`.

Reply:
I did a small mistake by writing the app_version_textview_margin_end twice.
I had corrected it now.